sample-components-prefab/src/main.js

```diff
@@ -0,0 +1,17 @@
+// Add support for things like setTimeout, setInterval and fetch.
+// Simply importing this sets all these as global definitions.
+// They are declared in the .eslintrc so your editor won't complain.
+import 'magic-script-polyfills';
+import process from './global-scope.js';
```

Comment:
Do we need to import `process` here?  It could be simply `import './global-scope.js';`

Reply:
done